src/commands/test_balances.rs

```diff
@@ -83,13 +100,35 @@ impl TransactionCreator for TransferInfo {
         let to_addr = wallet.address_for_account(self.acc_idx_to, External);
         create_transfer_transaction(from_addr, to_addr, self.amount, self.asset, wallet)
     }
+
+    fn create_tx_multi_user(&self, wallets: &mut [&mut User]) -> Transaction {
+        let from_addr = wallets[self.acc_idx_from].address_for_account(0, External);
+        let to_addr = wallets[self.acc_idx_to].address_for_account(0, External);
```

Comment:
Note that I have multiple `Users`, but I am using "account" 0 for them all. I don't support multiple accounts per `User` at the moment.